assets/css/screenplay-container.scss

```diff
@@ -10,6 +10,7 @@
     overflow-y: auto;
     overflow-x: hidden;
     color: $text-primary;
+    background-color: #0f1417;
```

Comment:
nit: Can this be put in `variables.scss`? I see it being used as a bg-color elsewhere so would be helpful to have it only defined once in case it changes again.